gradle/libs.versions.toml

```diff
@@ -72,7 +72,9 @@ koinCore = "3.5.3"
 koinAndroid = "3.5.3"
 koinAndroidCompose = "3.5.3"
 koinTest = "3.5.3"
-kotlin = "1.9.22"
+kotlinBom = "1.9.22"
+# Needs to match the version used by Gradle
```

Comment:
How does someone updating Gradle know which version to put here? Should we add a link to <https://docs.gradle.org/current/userguide/compatibility.html#kotlin> (although there's currently no entry specifically mentioning e.g. Gradle 8.6)?

Reply:
Just run  `./gradlew --version`, it will report the currently used Kotlin version.